README.md

```diff
@@ -47,6 +45,14 @@ This project also comes pre-configured to load in media files, which you can ach
 
 By default, supported media types are: png | jpg | gif | mp3 | ogg | mp4
 
+### Static Files
+
+In the case where you wish to include static files in your project, SpringRoll Seed has the option to automatically add them to your build. Place any static files in the `static` directory and they will be included to the root of your release directory with the same structure as in the `static` directory.
+
+These files will not be modified by Webpack's build process.
+
+During development you can access these files just as you would during production. You do not need to include the `static` directory as part of the file path.
```

Comment:
I declare this to be awesome. However, could you provide an example as well (even if it's just an HTML snippet or something)?